packages/dev/inspector-v2/src/components/properties/nodes/gaussianSplattingProperties.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FunctionComponent } from "react";
+
+import type { GaussianSplattingMesh } from "core/index";
+
+import { StringifiedPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/stringifiedPropertyLine";
+import { BooleanBadgePropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/booleanBadgePropertyLine";
+
+export const GaussianSplattingDisplayProperties: FunctionComponent<{ mesh: GaussianSplattingMesh }> = (props) => {
+    const { mesh } = props;
+
+    return (
+        <>
+            <StringifiedPropertyLine label="Splat Count" value={mesh.splatCount ?? 0} />
+            <StringifiedPropertyLine label="SH Parameter Count" value={(mesh.shDegree + 1) * (mesh.shDegree + 1) - 1} />
+            <BooleanBadgePropertyLine label="Has Compensation" value={mesh.compensation} />
+            <StringifiedPropertyLine label="Kernel Size" value={mesh.kernelSize} />

```

Comment:
Are any of these properties things that might change at runtime? If so wrapping them in BoundProperty would be good, otherwise this is fine.

Reply:
These properties come from the loaded splat file so readonly.